src/Conversion/KrnlToLLVM/KrnlGlobal.cpp

```diff
@@ -289,24 +289,42 @@ class KrnlGlobalOpLowering : public ConvertToLLVMPattern {
     Location loc = krnlGlobalOp.getLoc();
     MultiDialectBuilder<LLVMBuilder> create(builder, loc);
 
-    ModuleOp module = krnlGlobalOp->getParentOfType<ModuleOp>();
     DenseElementsAttr denseAttr =
         krnlGlobalOp.getValue().value().cast<DenseElementsAttr>();
 
     Type i8PtrType = getI8PointerType(builder.getContext());
 
-    // Generate LLVM GlobalOps for each string in the KrnlGlobalOp dense
-    // attribute.
-    SmallVector<LLVM::GlobalOp> globalOps;
-    for (StringRef str : denseAttr.getValues<StringRef>()) {
-      LLVM::GlobalOp globalOp = krnl::getOrCreateGlobalString(
-          str, loc, builder, module, getTypeConverter());
-      globalOps.push_back(globalOp);
+    auto strs = denseAttr.getValues<StringRef>();
+    // Collect total size of the strs.
+    size_t totalSize = 0;
+    for (StringRef str : strs) {
+      // Add 1 for the null terminator.
+      totalSize += str.size()+1;
```

Comment:
Looks like this caused the clang-format check failed. You may run `git clang-format` to format new code before pushing them.

Reply:
Fixed.